perfkitbenchmarker/linux_benchmarks/dpb_wordcount_benchmark.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the word count job on data processing backends.
+
+WordCount example reads text files and counts how often words occur. The input
+is text files and the output is text files, each line of which contains a word
+and the count of how often it occured, separated by a tab.
+"""
+
+import tempfile
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker.dpb_service import BaseDpbService
+from perfkitbenchmarker import flags
+from perfkitbenchmarker.providers.aws import aws_dpb_emr
+from perfkitbenchmarker.providers.gcp import gcp_dpb_dataproc
+from perfkitbenchmarker.providers.gcp import gcp_dpb_dataflow
+
+BENCHMARK_NAME = 'dpb_wordcount_benchmark'
+
+BENCHMARK_CONFIG = """
+dpb_wordcount_benchmark:
+  description: Run word count on dataflow and dataproc
+  flags:
+    dpb_wordcount_out_fs: gs
+    dpb_wordcount_out_base: saksena-df
+  dpb_service:
+    service_type: dataproc
+    worker_group:
+      vm_spec:
+        GCP:
+          machine_type: n1-standard-1
+          boot_disk_size: 500
+        AWS:
+          machine_type: m1.medium
+      disk_spec:
+        GCP:
+          disk_type: nodisk
+        AWS:
+          disk_size: 500
+          disk_type: gp2
```

Comment:
I can add comment to the description of the benchmark explaining how the values will be used